java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java

```diff
@@ -7048,8 +7217,8 @@ public Long scheduleSPMigration(User loggedInUser, Integer sid, String targetIde
         }
 
         // We didn't find target products if we are still here
-        throw new FaultException(-1, "servicePackMigrationNoTarget",
-                "No target found for SP migration");
+        throw new FaultException(-1, "productMigrationNoTarget",
+                "No target found for Product migration");
```

Comment:
I hope no customer scripts implemented error handling based on these values.

Reply:
If you say it can be accepted I will keep it like it is.
Let's just hope no customer uses these :P

Reply:
ok